docs/builder-guides/Use Astar Features/How to use Hardhat to deploy on Shibuya.md

```diff
@@ -0,0 +1,171 @@
+# How to use Hardhat to deploy on Shibuya
+
+## TL;DR
+
+Hardhat is an Ethereum (EVM) development environment that helps developers to manage and automate the recurring tasks inherent to the process of building smart contracts and dApps, which means compiling and testing at the very core. 
+
+Since Astar Network is a multi-VM smart contract hub, we support both WASM and EVM smart contracts, which means you can use Ethereum dev tools including Hardhat to directly interact with Astar EVM’s API and deploy Solidity smart contracts on Astar.
+
+In this cookbook, we will guide you on how to set up the environment for Astar EVM development, how to create and configure a Hardhat project for Astar EVM, and how to deploy a simple Solidity smart contract on Astar EVM via Hardhat.
+
+---
+
+## What is Astar EVM?
+
+As a multi-VM smart contract hub, Astar Network supports both WASM and EVM smart contracts, which means both Solidity smart contracts and WASM-based smart contracts can be deployed on Astar Network. 
+
+And for Solidity developers, you can directly use Ethereum dev tools including Hardhat, Remix, MetaMask to directly interact with Astar EVM’s API and deploy Solidity smart contracts on Astar EVM.
+
+## What is Hardhat?
+
+Hardhat is a development environment that helps developers in testing, compiling, deploying, and debugging smart contracts and dApps on Ethereum Virtual Machine (EVM). Hardhat smart contract development environment offers suitable tools to developers for managing the development workflow, and identifying causes for the failure of applications.
+
+---
+## Set up Node.js environment for Hardhat
+Hardhat is built on top of Node.js, the JavaScript runtime built on Chrome's V8 JavaScript engine. As the first step for to set up Hardhat, we need to set up Node.js environment.
+
+Make sure you have git installed and run the following command in the terminal.
+
+---
+## Create a Hardhat project
+
+First, let’s create a directory for this tutorial with the following command.
+
+```bash
+mkdir hardhat_test
+cd hardhat_test
+```
+
+
+Then, let’s initialize npm environment.
+
+```bash
+npm init -y
+```
+
+After the command above, you will see the following return message.
+
+```
+Wrote to /Users/suguruseo/Documents/Astar Work/hardhat/test-hardhat/hardhat_test/package.json:{
+  "name": "hardhat_test",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC"
+}
+```
+
+At last, we install Hardhat and create a Hardhat project.
+
+```bash
+npm install hardhat
+```
+
+![1](img/1.png)
+
+
+---
+### Set up a private key
+
+In this section, we will set up a private key.
+
+```bash
+touch private.json
+vim private.json
+```
+
+Then, we add the private key which Hardhat prepared as default.
```

Comment:
Please explain in more detail what "Hardhat prepared as default" means? Is this general private key for anybody in the world or specific to this installation (where do I find this key expect in astar docs)

I suggest you also share a link to this specific info withing Hardhat documentation

Reply:
I stopped at this point in the tutorial because I was not being able to follow anymore? (Did not know where to find this private key)

Reply:
@newnewsms fixed this issue.